### PR TITLE
Link "Configure integations" button to Communications tab

### PR DESCRIPTION
**Pull request by leSamo**
https://issues.redhat.com/browse/RHINENG-9423

### CODE REVIEW

File: src/SmartComponents/SystemInventory/SystemInventoryHeader.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ const SystemInventoryHeader = ({
                                     </FlexItem>
                                 </Flex>
                                 <FlexItem align={{ md: 'alignRight' }}>
-                                    <Link to="/settings/integrations">
+                                    <Link to="/settings/integration?category=Communications">
```
Comment (johnsonm325) [inline src/SmartComponents/SystemInventory/SystemInventoryHeader.js:133]:
```suggestion
                                    <Link to="/settings/integrations?category=Communications">
```

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
I think you left off the `s` by accident.

File: src/SmartComponents/SystemInventory/SystemInventoryHeader.js (hunk 1 above)
Comment (leSamo) [reply src/SmartComponents/SystemInventory/SystemInventoryHeader.js]: Good catch! You are right

**Review by mkholjuraev — APPROVED** [review]
LGTM!

**Pull request merged**